bundles/org.openhab.core.automation/src/main/java/org/openhab/core/automation/internal/module/handler/ChannelEventTriggerHandler.java

```diff
@@ -88,7 +88,7 @@ public boolean apply(Event event) {
         boolean eventMatches = false;
         if (event instanceof ChannelTriggeredEvent) {
             ChannelTriggeredEvent cte = (ChannelTriggeredEvent) event;
-            if (cte.getTopic().contains(this.channelUID)) {
+            if (cte.getTopic().equals("openhab/channels/" + this.channelUID + "/triggered")) {
```

Comment:
Why do we check against the topic at all? The `ChannelTriggeredEvent` stores the `ChannelUID` inside which can be used for comparison:

https://github.com/openhab/openhab-core/blob/0972ad5ede205db5ebbcde5e2c5bbe205bb94621/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/events/ChannelTriggeredEvent.java#L65-L70

Reply:
Good point, I'll change it.